App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
```

Comment:
Added import statement for Quotes component.

Reply:
you can remove this comment